src/requirements.txt

```diff
@@ -1,21 +1,17 @@
-django~=3.2.16
-djangorestframework~=3.11.0
-django-rest-swagger~=2.2.0
-drf-yasg~=1.17.0
-mozilla-django-oidc~=1.2.3
-whitenoise~=5.0.1
-dj-database-url~=0.5.0
-requests~=2.23.0
-django-watchman~=1.1.0
-django-webpack-loader~=0.6.0
-django-safedelete~=0.5.4
-jsonfield~=3.1.0
+django==3.2.18
+djangorestframework==3.14.0
+drf-spectacular==0.25.1
+mozilla-django-oidc==3.0.0
+whitenoise==6.3.0
+dj-database-url==1.2.0
+requests==2.28.2
+django-watchman==1.3.0
+django-webpack-loader==0.6.0
```

Comment:
This package will need an update when we work on the frontend. It needs to stay in sync with the companion `npm` package.

Reply:
Can we add this as a comment here? Or is it mentioned on the issue for the update for the frontned?

Reply:
It's changed in this commit from the FE update (https://github.com/tl-its-umich-edu/remote-office-hours-queue/pull/405/commits/4ebecb2511d1377afc21ed62d835a77ea55808e1), I'm keeping track of it, I dunno if a comment is really needed.